src/OMSimulatorLib/Connection.cpp

```diff
@@ -130,6 +148,16 @@ void oms3::Connection::setGeometry(const oms2::ssd::ConnectionGeometry* newGeome
   this->geometry = reinterpret_cast<ssd_connection_geometry_t*>(geometry_);
 }
 
+void oms3::Connection::setTLMParameters(double delay, double alpha, double impedance, double impedancerot)
+{
+  tlmparameters = new oms3_tlm_connection_parameters_t;
```

Comment:
Is it possible, that this method is called multiple times? It would probably be safer to check if `tlmparameters` was already allocated.